src/Bitcoin/CashAddress.h

```diff
@@ -15,6 +15,8 @@
 
 namespace TW::Bitcoin {
 
+static const std::string BITCOIN_CASH_HRP = "bitcoincash";
```

Comment:
If this is used by BitcoinCashAddress class, it's better to move inside that class.